src/hacking/building-for-android.md

```diff
@@ -15,7 +15,7 @@ Versions before and after change the layout of the NDK and add or remove files.
 
 If you are using the `sdkmanager` tool, you can do:
 ```sh
-tools/bin/sdkmanager platform-tools "platforms;android-33" "build-tools;34.0.0" "ndk;26.2.11394342"
```

Comment:
`sudo` is not required unless your SDK root is in a path that is not accessible to the user that is running the `sdkmanager`. The [official guide for installing command-line tools](https://developer.android.com/tools/sdkmanager) also assumes that the SDK root is inside the user's home directory and they have access to it, so it doesn't suggest using `sudo` when running `sdkmanager`.